source/keyLabels.py

```diff
@@ -162,3 +162,6 @@
 	# Translators: This is the name of a key on the keyboard.
 	'tab': pgettext("keyLabel", "tab"),
 }
+
+def getKeyCombinationLabel(keyCombination):

```

Comment:
Done.
Furthermore, I noticed that my code failed if a particular key wasn't in localizedKeyLabels. When that happens now, the code falls back to the lowercase version of the originally provided key. Yay for list comprehension power!